daml/Passage.daml

```diff
@@ -0,0 +1,38 @@
+module Passage where
+
+template LocalPassage with
+    id: Int
+    master: Party
+    issuer: Party
+    citizenId: Int 
+    club: Text
+    passageDate: Time
+    permitId: Int
+  where
+    signatory issuer
+    observer master
+    key (id, issuer) : (Int, Party)
```

Comment:
That means we need to take that concatenation between id and issuer fields will be unique, given id provided by the user, are we ok to push this logic into application level?

Reply:
The application is responsible for making sure that the ID is unique with respect to all of the IDs that are currently visible to it. However, conflicts could still happen - for example, two secondary servers can each create a `LocalPassage` contract using the same ID within a short time window (because they weren't synchronized), which would result in a conflict. We are aware of this, but conflict resolution is not within the scope of this PR.